MavenQuickStart/quickApp/src/main/java/com/globant/topicfive/Main.java

```diff
@@ -13,13 +14,16 @@ public static void main(String args[]) {
 		try {
 			// To connect to mongodb server
 			MongoClient mongoClient = new MongoClient("localhost", 27017);
-			DB database = mongoClient.getDB("HighSchool");
-			DBCollection coll = (DBCollection) database.getCollection("highSchoolCollection");		
+			DB database = mongoClient.getDB("high-school");			
+			DBCollection student = (DBCollection) database.getCollection("highSchool");
```

Comment:
Please try to use the same collection names in java than in the DB, otherwise it becomes hard to follow.